### PR TITLE
Update reserved-capacity-overview.md

### PR DESCRIPTION
**Pull request by KoprowskiT**
Minor Grammar and wording changes

### CODE REVIEW

**Comment by Court72** [comment]
@rajeshsetlem

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-sql/database/reserved-capacity-overview.md [hunk 1]
```diff
@@ -17,9 +17,9 @@ monikerRange: "=azuresql||=azuresql-db||=azuresql-mi"
...
+You do not need to assign the reservation to a specific database or managed instance. Matching existing deployments that are already running or ones that are newly deployed automatically gets the benefit. Hence, by purchasing a reserved capacity, existing resources infrastructure would not be modified and thus no failover/downtime is triggered on existing resources. By purchasing a reservation, you commit to usage for the compute costs for a period of one or three years. As soon as you buy a reservation, the compute charges that match the reservation attributes are no longer charged at the pay-as-you-go rates.
```
Comment (rwestMSFT) [inline azure-sql/database/reserved-capacity-overview.md]:
```suggestion
You do not need to assign the reservation to a specific database or managed instance. Matching existing deployments that are already running, or ones that are newly deployed automatically, get the benefit. Hence, by purchasing a reserved capacity, existing resources infrastructure would not be modified and thus no failover/downtime is triggered on existing resources. By purchasing a reservation, you commit to usage for the compute costs for a period of one or three years. As soon as you buy a reservation, the compute charges that match the reservation attributes are no longer charged at the pay-as-you-go rates.
```

File: azure-sql/database/reserved-capacity-overview.md [hunk 2]
```diff
@@ -31,9 +31,9 @@ For more information about how enterprise customers and Pay-As-You-Go customers
 > [!NOTE]
 > Purchasing reserved capacity does not pre-allocate or reserve specific infrastructure resources (virtual machines or nodes) for your use.
 
-## Determine correct size before purchase
+## Determine the correct size before purchase
 
-The size of reservation should be based on the total amount of compute used by the existing or soon-to-be-deployed database or managed instance within a specific region and using the same performance tier and hardware configuration.
+The size of the reservation should be based on the total amount of compute used by the existing or soon-to-be-deployed database or managed instance within a specific region and using the same performance tier and hardware configuration.
```
Comment (rwestMSFT) [inline azure-sql/database/reserved-capacity-overview.md]:
```suggestion
The size of reservation should be based on the total amount of compute used by the existing or soon-to-be-deployed database or managed instance within a specific region and using the same performance tier and hardware configuration.
```

**Comment by rwestMSFT** [comment]
The first "gets" was an incorrect edit. I undid it before merging.

**Pull request merged**